dist/modules/sound.js

```diff
@@ -446,6 +446,7 @@
 		Debug,
 		SoundContext,
 		SoundInstance,
+		CordovaAudioPlugin = include('createjs.CordovaAudioPlugin'),
```

Comment:
Yeah, I just had to change this line to `CordovaAudioPlugin = include('createjs.CordovaAudioPlugin', false),` and also bump the version in bower to take the latest SoundJS tag.